### PR TITLE
Increase coverage

### PR DESCRIPTION
**Pull request by dancerphil**
## Checklist
- [x] test App with viewType === 'unified'
- [x] test hocs with basic usage
- [x] test tokenize, markEdits, markWord, pickRanges
- [x] test Diff with Decoration (split and unified)
- [x] test hocs with usage in [document](https://github.com/otakustay/react-diff-view/blob/6588a2fb8a0100e0ff365b60caa03bdff4050c87/docs/hoc.md)
- [x] test Widget (split and unified)
- [x] test CodeCell
- [x] test utils
- [x] add travis
- [x] delete unused code in `src/tokenize/utils.js`

Merge this after list is all checked.

---

## Further Plan
- enzyme is necessary for further test.
- split `src/utils/diff.js` into several files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Here is a very basic example which adds a warning text on long lines:
 ```javascript
 import {parseDiff, getChangeKey, Diff} from 'react-diff-view';
 
-const getWidgets = ({hunks}) => {
+const getWidgets = hunks => {
```
Comment (dancerphil) [inline README.md]: README 修正

File: docs/hoc.md [hunk 2]
```diff
@@ -78,18 +78,18 @@ As an example, we add a decoration between hunks and allow user to expand the co
...
     return (
         <div onClick={() => onClick(start, end)}>
             Click to expand
         </div>
-    )
+    );
 };
 
-const renderHunk = (children, hunk, i, hunks) => {
+const renderHunk = onExpandRange => (children, hunk, i, hunks) => {
```
Comment (dancerphil) [inline docs/hoc.md]: 示例有问题，onExpandRange 不在函数域

File: src/hocs/__test__/withChangeSelect.test.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    };
+    const renderHunk = hunk => (
+        <Hunk
+            key={hunk.content}
+            hunk={hunk}
+            codeEvents={codeEvents}
+        />
+    );
+
+    // TODO modify document
```
Comment (dancerphil) [inline src/hocs/__test__/withChangeSelect.test.js]: diffType 和 viewType 在代码里是 required，但是在文档里没有写。

File: src/tokenize/utils.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ export const split = (path, splitStart, splitEnd, wrapSplitNode) => {
     return output;
 };
 
+// TODO the func is unused, make sure it is useless and delete it
```
Comment (dancerphil) [inline src/tokenize/utils.js]: 这段代码既没有被使用，也没有 export 到 package 之外。

**Comment by otakustay** [comment]
CI failed only for low coverage, will merge this first

**Pull request merged**